tck/features/OrderByAcceptance.feature

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright 2016 "Neo Technology",
+# Network Engine for Objects in Lund AB (http://neotechnology.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Feature: OrderByAcceptance
+
+  Background:
+    Given any graph
+
+  Scenario: ORDER BY should return results in ascending order
+    And having executed: CREATE (n1 {prop: 1}), (n2 {prop: 3}), (n3 {prop: -5})
+    When executing query: MATCH (n) RETURN n.prop AS prop ORDER BY n.prop
+    Then the result should be:
```

Comment:
If the query contains `ORDER BY`, the results expectations must match this, by saying `Then the results should be, in order:`, just like here: https://github.com/opencypher/openCypher/blob/master/tck/features/ReturnAcceptance.feature#L33

I see that this is not reflected at all in the readme currently. Please add something about it there as well. There is a section mentioning `ORDER BY`, but it doesn't state this requirement.